feature-pack/pom.xml

```diff
@@ -332,17 +332,6 @@
             <artifactId>jakarta.security.enterprise</artifactId>
         </dependency>
 
-        <dependency>
```

Comment:
If these artifacts are removed from feature-pack, they're not runtime artifacts anymore.
Moving them to galleon-pack means that they're build dependencies, not runtime.
Is it correct from the perspective of using WildFly Server?
cc @rgdoliveira

Reply:
They are still runtime dependencies of any server that uses the 'wildfly' galleon pack. The output of this maven module and the 'servlet-feature-pack' maven module are 'legacy feature packs' and are not used in any WildFly distribution. We just maintain these modules so projects that still want to create custom WF-technology-based servers using the legacy feature packs can use their output. But an official WildFly dist does not use the output of these. (You could do 'mvn install -pl dist -am' and this maven module wouldn't even build.) 

However, our official dists do use some *source* from these maven modules. This is so we can build the official Galleon feature packs and these legacy ones and not have to duplicate the source, which would almost certainly lead to bugs.  The Galleon feature pack poms actually copy over the source resources they need:

https://github.com/wildfly/wildfly/blob/master/galleon-pack/pom.xml#L73
https://github.com/wildfly/wildfly/blob/master/servlet-galleon-pack/pom.xml#L548